homeassistant/components/fitbit/const.py

```diff
@@ -0,0 +1,143 @@
+"""Constants for the Fitbit platform."""
+from __future__ import annotations
+
+import datetime
+from typing import Final
+
+from homeassistant.const import (
+    CONF_CLIENT_ID,
+    CONF_CLIENT_SECRET,
+    LENGTH_FEET,
+    MASS_KILOGRAMS,
+    MASS_MILLIGRAMS,
+    PERCENTAGE,
+    TIME_MILLISECONDS,
+    TIME_MINUTES,
+)
+
+_CONFIGURING: Final[dict] = {}
+
+ATTR_ACCESS_TOKEN: Final = "access_token"
+ATTR_REFRESH_TOKEN: Final = "refresh_token"
+ATTR_LAST_SAVED_AT: Final = "last_saved_at"
+
+CONF_MONITORED_RESOURCES: Final = "monitored_resources"
+CONF_CLOCK_FORMAT: Final = "clock_format"
+ATTRIBUTION: Final = "Data provided by Fitbit.com"
+
+FITBIT_AUTH_CALLBACK_PATH: Final = "/api/fitbit/callback"
+FITBIT_AUTH_START: Final = "/api/fitbit"
+FITBIT_CONFIG_FILE: Final = "fitbit.conf"
+FITBIT_DEFAULT_RESOURCES: Final = ["activities/steps"]
```

Comment:
```suggestion
FITBIT_DEFAULT_RESOURCES: Final[list[str]] = ["activities/steps"]
```

Reply:
it is really time to go to :sleeping_bed:   :see_no_evil: